pkg/api/unversioned/time.go

```diff
@@ -20,6 +20,9 @@ import (
 	"encoding/json"
 	"time"
 
+	"k8s.io/kubernetes/cmd/libs/go2idl/openapi-gen/generators"
```

Comment:
I think we should not have a dependency on the generator package. Can we move the OpenAPIDefinition type into a 3rd package that both the generator and this can depend on? It's fine if it lives as a subdirectory in the generator. I just don't want to pull in all of the generator's dependencies here.